packages/base/src/utils.ts

```diff
@@ -100,18 +100,25 @@ export function reject(message: string, log: boolean) {
 export function put_buffers(
   state: Dict<BufferJSON>,
   buffer_paths: (string | number)[][],
-  buffers: DataView[]
+  buffers: any[]
```

Comment:
`any` seems overly permissive. AFAICT, this should be a union of:
- `DataView`
- `ArrayBuffer`
- a generic type that includes a `buffer` attribute of the correct type (`ArrayBuffer`). This would catch TypedArrays.

Reply:
Thanks, we also need to add `ArrayBufferView` in the union.